x-pack/test/functional/apps/observability_logs_explorer/flyout.ts

```diff
@@ -65,32 +65,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       }
     });
 
-    it('should mount the flyout customization content', async () => {
+    it('should display the logs overview tab', async () => {
       await dataGrid.clickRowToggle({ columnIndex: 4 });
-      await testSubjects.existOrFail('logsExplorerFlyoutDetail');
-    });
-
-    it('should display a timestamp badge', async () => {
```

Comment:
Should these tests have been removed? I see this task is completed in the PR summary, but they don't seem to have been ported anywhere:
>Port the existing e2e tests for the logs overview tab into the unified-doc-viewer plugin.

Personally I'd be fine with leaving hem here with updated `data-test-subj` values until we can test them in Discover once enabled.

Reply:
These tests have been moved and adapted next to the doc view implementation: https://github.com/elastic/kibana/pull/180262/files#diff-35dde01753f22a9d917cb933594571e1d9583419455c0a00223e313bd2dd32a6